dev-perl/Sys-Virt/files/Sys-Virt-8.4.0-lib-Fix-parameter-detection-for-save-restore_domain.patch

```diff
@@ -0,0 +1,48 @@
+From be960176a8b045d54ac1dfa0cb76cef8ed9cc660 Mon Sep 17 00:00:00 2001
```

Comment:
Nice, thanks! For future, can you include a link to the relevant commit(s)?

Reply:
I'm not exactly sure I understand. Where exactly do you want me to put the link? And thanks for merging!